src/commands/config/create.ts

```diff
@@ -0,0 +1,211 @@
+        gasPriceScale: 1,
+        dryRun: false,
+      };
+      const { contractAddress } = await deployTokenRegistry(deployTokenRegistryParams);
+      return contractAddress;
+    };
+
+    const createTemporaryDnsWithTokenRegistry = async (): Promise<string> => {
+      tokenRegistryAddress = await createTokenRegistry();
+      success(`Token registry deployed, address: ${highlight(tokenRegistryAddress)}`);
+      info(`Creating temporary DNS for transferable records`);
+      const tokenRegistryTemporaryDnsParams = {
+        networkId: 3,
+        address: tokenRegistryAddress,
+        sandboxEndpoint: sandboxEndpointUrl,
+      };
+      return (await createTemporaryDns(tokenRegistryTemporaryDnsParams)) || "";
+    };
+
+    const updatedForms: Form[] = [];
+    for (const form of formsInTemplate) {
+      let updatedIssuers: Promise<Issuer>[] = [];
+      const updatedForm = form;
+      if (form.type === "VERIFIABLE_DOCUMENT") {
+        updatedIssuers = form.defaults.issuers.map(async (issuer) => {
+          if (issuer.identityProof?.type === "DNS-TXT") {
+            if (!verifiableDocumentDnsTxtName)
+              verifiableDocumentDnsTxtName = await createTemporaryDnsWithDocumentStore();
+            issuer.name = "DEMO STORE";
+            issuer.documentStore = documentStoreAddress;
+            issuer.identityProof.location = verifiableDocumentDnsTxtName;
+          } else if (issuer.identityProof?.type === "DNS-DID") {
+            if (!verifiableDocumentDnsDidName) verifiableDocumentDnsDidName = await createTemporaryDnsWithDid();
+            issuer.name = "Demo Issuer";
+            issuer.id = `did:ethr:0x${walletObject.address}`;
+            if (issuer.revocation?.type) issuer.revocation.type = "NONE" as RevocationType;
+            issuer.identityProof.location = verifiableDocumentDnsDidName;
+            issuer.identityProof.key = `did:ethr:0x${walletObject.address}#controller`;
+          }
+          return issuer;
+        });
+      }
+      if (form.type === "TRANSFERABLE_RECORD") {
+        updatedIssuers = form.defaults.issuers.map(async (issuer: Issuer) => {
+          if (!tokenRegistryDnsName) tokenRegistryDnsName = await createTemporaryDnsWithTokenRegistry();
+          issuer.name = "DEMO TOKEN REGISTRY";
+          issuer.tokenRegistry = tokenRegistryAddress;
+          if (issuer.identityProof?.location) issuer.identityProof.location = tokenRegistryDnsName;
+          return issuer;
+        });
+      }
+      const awaitedUpdatedIssuers = await Promise.all(updatedIssuers);
+      updatedForm.defaults.issuers = awaitedUpdatedIssuers;
+      updatedForms.push(updatedForm);
+    }
+
+    configFile.forms = updatedForms;
+    fs.writeFileSync(`${args.outputDir}/config.json`, JSON.stringify(configFile, null, 2));
+    success(`Config file successfully generated`);
+  } catch (e) {
+    error(e.message);
+  }
+};
+
+const whichFileToUseAsTemplate = (args: CreateConfigCommand): string => {
```

Comment:
suggestion to rename to selectTemplatePath or something similar

Reply:
ok